pandas/plotting/_matplotlib/core.py

```diff
@@ -383,6 +386,21 @@ def result(self):
             else:
                 return self.axes[0]
 
+    def _convert_to_ndarray(self, data):
+        # data = self.data
```

Comment:
Please delete this comment.

Reply:
Thanks for pointing this out @ivanovmg. 
Deleted the comment in the latest commit.